Build fails with fault in Datastore

Since merging the related PR the building fails on CI:
@moving-bits 


integration/main/src/cgeo/geocaching/storage/DataStore.java:3784: error: cannot find symbol
13:08:29  
13:08:29                  statement.bindString    (1, InternalConnector.GEOCODE_HISTORY_CACHE);  // geocode
13:08:29                                              ^
13:08:29    symbol:   variable InternalConnector
13:08:29    location: class DataStore

--- a/main/src/cgeo/geocaching/storage/DataStore.java
+++ b/main/src/cgeo/geocaching/storage/DataStore.java
@@ -6,6 +6,7 @@ import cgeo.geocaching.R;
 import cgeo.geocaching.SearchResult;
 import cgeo.geocaching.connector.IConnector;
 import cgeo.geocaching.connector.gc.Tile;
+import cgeo.geocaching.connector.internal.InternalConnector;
 import cgeo.geocaching.enumerations.CacheSize;
 import cgeo.geocaching.enumerations.CacheType;
 import cgeo.geocaching.enumerations.LoadFlags;

